test/jasmine/tests/sankey_test.js

```diff
@@ -433,10 +436,14 @@ describe('sankey tests', function() {
                 );
 
                 return Plotly.restyle(gd, {
-                    'hoverlabel.bgcolor': 'red',
-                    'hoverlabel.bordercolor': 'blue',
-                    'hoverlabel.font.size': 20,
-                    'hoverlabel.font.color': 'black'
+                    'node.hoverlabel.bgcolor': 'red',
+                    'node.hoverlabel.bordercolor': 'blue',
+                    'node.hoverlabel.font.size': 20,
+                    'node.hoverlabel.font.color': 'black',
+                    'link.hoverlabel.bgcolor': 'red',
+                    'link.hoverlabel.bordercolor': 'blue',
+                    'link.hoverlabel.font.size': 20,
+                    'link.hoverlabel.font.color': 'black'
```

Comment:
Lets use different values for node & link, just for good measure.

Reply:
Thanks. Lets let the reviewer mark a conversation resolved though, even if it's a simple one like this. Makes it a lot easier to tell what I need to look at to ensure all my comments have been addressed.

Reply:
Makes total sense! Sorry about that